### DISCUSSION TITLE
[vite]: Rollup failed to resolve import

### DISCUSSION THREAD

**Discussion by cnlancehu** [vuepress/core#998] [Q&A] [answered]
error:
`[vite]: Rollup failed to resolve import "/img/emotions/UI_EmotionIcon1.png" from "docs/.vuepress/.temp/pages/file-interpretation/extend-emotion.html.vue".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to build.rollupOptions.external`

how can i add the module to build.rollupOptions.external

**Comment by Mister-Hope** [comment, marked as answer]
That image does not exisit in public folder, my suggestions are add one or use full link.

Related docs are at: https://v2.vuepress.vuejs.org/reference/bundler/vite.html#viteoptions
